doc/guides/dts/index.rst

```diff
@@ -374,8 +378,8 @@ value or the format of its name.  Node properties are generated as C-style
 ``#define``'s in include files
 made available to all Zephyr components.
 
-A YAML template file is provided to show the required format.  This file is
-located at::
+The binding format is documented in this file, which also has detailed
```

Comment:
Agreed that the template should  be kept cleaner and most of the detail documentation should be put in a new document here, maybe ``dts_binding_format.rst``, or as an include file (as was done for flash partitions).

Reply:
I integrated it into the existing documentation.

The template is really the meat of the documentation in the sense that you have to read it to make sense of bindings, so it felt a bit weird to hide it behind a link. It's now included directly in the generated documentation.

Reply:
@dbkinder ping